sdk/schemaregistry/schema-registry-avro/samples-dev/schemaRegistryAvroSample.ts

```diff
@@ -69,7 +69,7 @@ export async function main() {
   console.log(buffer);
 
   // deserialize the result back to an object
-  const deserializedValue = await serializer.deserialize<User>(buffer);
+  const deserializedValue = (await serializer.deserialize(buffer)) as User;
```

Comment:
I'm fully OK with this, but I remember @xirzec originally suggested the generic approach. I originally had `any` which was definitely worse than both the others. Still, IIRC, we considered `unknown` and decided generic deliberately (I forget the rationale), so just making sure @xirzec had a chance to weigh in on this again.

Reply:
I think it's mostly just stylistic, I prefer to avoid needing the parens, but I don't have strong feelings if we'd prefer to model as 'unknown' and let the consumer decide what to do about it

Reply:
Besides, if you don't specify a value for the generic, it's going to default to `unknown` anyway

Reply:
Thanks @nguerrera and @xirzec for the context! I will go with `unknown` for now but this will surely be revisited again anyway in the archboard.